.gitignore

```diff
@@ -4,6 +4,7 @@ __pycache__
 *.pyc
 *.[aox]
 *.mod
+*.sw[a-p]
```

Comment:
Does emacs have similar temporary files we should ignore?

Reply:
I don't know about `emacs`.  I know some IDE's create `filename~`